manifests/z/Zanvok/Browsio/98.21.12.26/Zanvok.Browsio.locale.en-IN.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Zanvok.Browsio
+PackageVersion: 98.21.12.26
+PackageLocale: en-IN
+Publisher: Zanvok
+PublisherUrl: https://sites.google.com/view/zanvokcorporation
+PublisherSupportUrl: https://sites.google.com/view/zanvokhelp
+Author: Gautham Nair
+PackageName: Browsio
+PackageUrl: https://sites.google.com/view/browsio
+License: MPL-2.0 License
+Copyright: 2022 Zanvok Corporation
+ShortDescription: A Simple & Useful Tabbed Browser written in Python3
+Description: A Simple & Useful Tabbed browser written in Python3. This browser support all the scripts and websites, but lacks in few features and is only intended to run for emergency purposed, as it has more speed than other browsers.
+Moniker: Browser
+Tags:
+- Browser Zanvok Browsio Python Python3

```

```suggestion
- Browser
- Python
- Python3
```

Tags are arrays in yaml, which is done line-by-line with `-`. Zanvok and Browsio aren't necessary as tags because they are covered by the `Publisher` and the `PackageName`